vars/mavenNode.groovy

```diff
@@ -5,6 +5,19 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def file = new File("/var/lib/jenkins/nodes")
+    
+    if(!file.exists()){
+        error "file or directory /var/lib/jenkins/nodes doesnot exist"
+    } else {
+        if(!file.isDirectory()){
+            error "/var/lib/jenkins/nodes exists, but it is not a directory"
+        } else {
```

Comment:
else is not needed

Reply:
That's a sanity check to make sure that things are good. I would rather have it.

Reply:
removing the ` else {` and thereby the nested `if ` the code will work just as before, won't it?